app/markets/Bet.tsx

```diff
@@ -0,0 +1,72 @@
+'use client';
+
+import { ButtonLink, Icon, Tag } from '@swapr/ui';
+import { Market } from '@/entities';
+import { FixedProductMarketMaker } from '@/queries/omen';
+import { Swapbox } from '@/app/components';
+import { KLEROS_URL, REALITY_URL } from '@/constants';
+import { isPast } from 'date-fns';
+
+export const Bet = ({ market }: { market: FixedProductMarketMaker }) => {
+  const marketModel = new Market(market);
+
+  if (!marketModel.isClosed && marketModel.hasLiquidity)
+    return <Swapbox market={market} />;
+
+  if (!market.isPendingArbitration && marketModel.answer === null && marketModel.isClosed)
+    return (
+      <div className="flex flex-col items-center space-y-6 p-8">
+        <p className="text-md">
+          Final answer still <span className="font-semibold">pending</span>.
+        </p>
+        {market.question && (
+          <ButtonLink
+            href={`${REALITY_URL}/app/#!/network/100/question/0x79e32ae03fb27b07c89c0c568f80287c01ca2e57-${market.question.id}`}
+            target="_blank"
+            width="fit"
+            className="flex items-center space-x-1"
+            variant="ghost"
+          >
+            <span>Submit answer</span>
+            <Icon name="arrow-top-right" size={16} />
+          </ButtonLink>
+        )}
+      </div>
+    );
+
+  if (market.isPendingArbitration)
+    return (
+      <div className="flex flex-col items-center space-y-6 p-8">
+        <p className="text-md">
+          Final answer is being <span className="font-semibold">disputed</span>.
```

Comment:
WDYT about this wording? Makes sense?

```suggestion
          The result is being <span className="font-semibold">disputed</span>.
```

Reply:
I like it, updated!